README.md

```diff
@@ -37,7 +37,7 @@ Lead Maintainer - [Colin Ihrig](https://github.com/cjihrig)
         - [`exist()`](#exist)
         - [`empty()`](#empty)
         - [`length(size)`](#lengthsize)
-        - [`equal(value)`](#equalvalue)
+        - [`equal(value)`](#equalvalue-options)
```

Comment:
Can you update this to also include the `options` arg in the function signature.